bionic/flow.py

```diff
@@ -577,6 +579,7 @@ def full_name(first_name, last_name):
         state = self._state
 
         names = [name for name, value in name_value_pairs]
+        print(names)
```

Comment:
remove print statement. Maybe we should add a flake8 check for print statements too.

Reply:
Great idea. It looks like [flake8-print](https://pypi.org/project/flake8-print/) does this -- I'll see if I can set it up. (I also want to check out [flake8-fixme](https://pypi.org/project/flake8-fixme/), which addresses another common annoyance of mine.)